datasafe-business/pom.xml

```diff
@@ -184,6 +184,12 @@
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.15.4</version>
```

Comment:
why version 2.15.4. Try to use ${jackson.version} variable. It's defined in parent pom.